lib/container/RemoteRuntimeModule.js

```diff
@@ -35,20 +34,13 @@ class RemoteRuntimeModule extends RuntimeModule {
 				const module = /** @type {RemoteModule} */ (m);
 				const name = module.internalRequest;
```

Comment:
The files I am exposing from remotes were getting prefixed with `'./'` so I had to strip it off to avoid the `Container missing` error:

```javascript
let name = module.internalRequest;
name = name.startsWith("./") ? name.slice(2) : name;
```

I know this isn't the proper fix, but I couldn't figure out how to update name properly 🤷‍♂️

Reply:
None of the keys in `idToExternalAndNameMapping` have that `./` prefix so the lookup was failing.

Reply:
`./` is replaced with `webpack/container/restofquery`

Reply:
This is related to a previous PR: https://github.com/webpack/webpack/pull/10931

Reply:
Thanks! It's fixed on master branch now.